public-images/public-images-how-to/launch-with-libvirt.rst

```diff
@@ -0,0 +1,147 @@
+.. _launch-libvirt:
+
+Launch QCOW images using libvirt
+================================
+
+Amongst the virtualization tools available for Ubuntu, there is `libvirt`_. `libvirt`_ is not a hypervisor, but rather a set of tools and libraries to manage various other hypervisors, such as `KVM`_ and `QEMU`_.
+
+
+Although there are many more `applications using libvirt`_, this guide will focus on common ones. Those can be installed with of the following command:
+
+.. code:: bash
+
+   # graphical client
+   sudo apt-get install virt-manager
+
+
+The *recommended packages* for the aforementioned package installations will also install the `QEMU`_ libvirt driver, along with the command-line programs which will be used in this guide.
```

Comment:
"driver" here and below when discussing QEMU may not be the best word. Isn't "hypervisor" more accurate? Not a blocker, just a thought.

Reply:
both, but as the packaging currently refers to them as drivers (i.e. libvirt-daemon-driver-qemu), I used the same terminology.